.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore compilation artifacts
+**/target
+
+# Ignore staging temporary space
+tmp/
+
+# Ignore staged files
+**/.windows/lib
+**/.windows/winmd
```

Comment:
Typically the dependencies will get committed so that anyone can `git clone` and `cargo build` without chasing down dependencies. If I try that with this branch I get a failure. Not sure if its related to this though.

Reply:
```
C:\git\project-reunion-rs>git checkout users/riverar/initial
Switched to a new branch 'users/riverar/initial'
Branch 'users/riverar/initial' set up to track remote branch 'users/riverar/initial' from 'origin'.

C:\git\project-reunion-rs>cargo build
   Compiling proc-macro2 v1.0.27
   Compiling unicode-xid v0.2.2
   Compiling syn v1.0.73
   Compiling windows_gen v0.11.0
   Compiling const-sha1 v0.2.0
   Compiling project-reunion-rs v0.0.0 (C:\git\project-reunion-rs)
error: failed to run custom build command for `project-reunion-rs v0.0.0 (C:\git\project-reunion-rs)`

Caused by:
  process didn't exit successfully: `C:\git\project-reunion-rs\target\debug\build\project-reunion-rs-7e919ebe47c8e27c\build-script-build` (exit code: 101)
  --- stdout
  cargo:rustc-link-search=native=.windows/lib/x86_64

  --- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 3, kind: NotFound, message: "The system cannot find the path specified." }', build.rs:14:32
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
```

Reply:
Rather than commit large binary deps, I currently rely on devs to run `stage.ps1` before build. Not ideal or idiomatic Rust, I hear ya.